JoyboyCommunity/src/shared/components/Post.tsx

```diff
@@ -62,84 +168,111 @@ export const Post: React.FC<PostProps> = (props) => {
   const handleNavigateToPostDetails = () => {
     navigation.navigate('PostDetail', {postId: event?.id, post: event});
   };
+
   /** @TODO react in Nostr */
   const handleReact = () => {};
 
+  /** @TODO comment in Nostr */
+  const toggleLike = () => {
+    setIsLiked((prev) => !prev);
+    setLikes((prev) => {
+      const newLikes = isLiked ? prev - 1 : prev + 1;
+      textOpacity.value = newLikes > 0 ? 1 : 0;
+      iconPosition.value = newLikes > 0 ? 0 : 6;
+      return newLikes;
+    });
+    scale.value = withSequence(
+      withTiming(1.5, {duration: 25, easing: Easing.out(Easing.ease)}),
+      withSpring(1, {damping: 6, stiffness: 200}), // Small rebound effect
+    );
+  };
+
+  useEffect(() => {
+    if (likes > 0) setIsLiked(true);
```

Comment:
This should be removed since this does nothing.

Reply:
i realised we actually dont need two states for this, my bad. will update and simplify the logic here

Reply:
haha my bad